### PR TITLE
Upgrade to Bot Framework v4 SDK and ES6

### PR DESCRIPTION
**Pull request by adlele**
Upgraded to ES6
Upgraded to Bot Framework v4
Added hot reloading

### CODE REVIEW

File: src/tabs.js [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
+import fs from 'fs'
+
+export default function tabs(server) {
+
+    var staticBasePath = 'src/views/';
     // Setup home page
-    app.get('/', function(req, res) {
-        res.render('hello');
+    server.get('/', (req, res, next) => {
+        fs.readFile(staticBasePath + 'hello.html', 'utf8', function(err, data) {
```
Comment (aosolis) [inline src/tabs.js:8]: Let's simplify this by using the send package: https://github.com/pillarjs/send (https://www.npmjs.com/package/send)

File: src/views/configure.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="panel">
+      <div class="font-semibold font-title">Configure your app here
+        <p>
+          <div><label for="tabChoice"></label>Select the tab you would like to see:
+              <select id="tabChoice" name="tabChoice">
+                  <option value="" selected="selected">(Select a tab)</option>
+                  <option value="first">First</option>
+                  <option value="second">Second</option>
+              </select></div>
+      </p>
```
[the remark this answers is not in the document]
Comment (aosolis) [reply src/views/configure.html:29]: Let's reformat the HTML files for consistent indentation

File: package.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "msteams-nodejs-hello-world",
   "version": "1.0.0",
-  "description": "Microsoft Teams Node.js Hello World App. Start here to build your first app on the Teams platform.",
+  "description": "Microsoft Teams hello world sample app in written in ES6 with bot framework v4",
```
[the remark this answers is not in the document]
Comment (adlele) [reply package.json]: done

File: src/views/configure.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<head>
+  <title>
+    Microsoft Teams Hello World Sample App
+  </title>
+  <link rel="stylesheet" type="text/css" href="/styles/msteams-16.css">
+  </link>
+  <link rel="stylesheet" type="text/css" href="/styles/custom.css">
+  </link>
+  <script src="https://unpkg.com/@microsoft/teams-js@1.3.7/dist/MicrosoftTeams.min.js"
```
[the remark this answers is not in the document]
Comment (adlele) [reply src/views/configure.html]: done

File: src/tabs.js (hunk 1 above)
Comment (adlele) [reply src/tabs.js]: done

File: src/views/configure.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (adlele) [reply src/views/configure.html]: done

**Pull request merged**